data/json/items/comestibles/wheat.json

```diff
@@ -36,6 +37,7 @@
     "calories": 667,
     "description": "Raw wheat, not very tasty.",
     "price": 120,
+    "phase": "powder",
```

Comment:
This will break wheat farming. The player will harvest the mature crop and end up with material they cannot interact with.